File: benchmarks/py/fib_recursive.py
import sys


def fib(n: int) -> int:
  if n <= 1:
    return n
  else:
    return fib(n - 1) + fib(n - 2)


if __name__ == "__main__":
  n = int(sys.argv[1]) if len(sys.argv) > 1 else 0
  print(fib(n))

File: benchmarks/js/fib_recursive.js
function fib(n) {
  if (n <= 1) {
    return n;
  } else {
    return fib(n - 1) + fib(n - 2);
  }
}

const n = process.argv[2] ? parseInt(process.argv[2]) : 0;
console.log(fib(n));
